lib/presentation/widgets/custom_button.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:askm/presentation/tokens/spacing.dart';
+import 'package:flutter/material.dart';
+
+class CustomButton extends StatelessWidget {
```

Comment:
i dont like this widget with a lot of fields, you need two type of button, how i was said before, also add prefix app name like ASKMElevatedButoon and ASKMOutlinedButoon